spec/services/merge_patient_service_spec.rb

```diff
@@ -0,0 +1,95 @@
+require "rails_helper"
+
+RSpec.describe MergePatientService, type: :model do
```

Comment:
This class is doing a lot without any tests, so we've written tests for most of its behavior since it was scary to change it otherwise.

Reply:
✨ ✨ ✨